Source/Particles/ParticleIO.H

```diff
@@ -92,4 +92,7 @@ void
 storePhiOnParticles ( PinnedMemoryParticleContainer& tmp,
     ElectrostaticSolverAlgo electrostatic_solver_id, bool is_full_diagnostic );
 
+void
+storeEMFieldsOnParticles (PinnedMemoryParticleContainer& tmp,
+    ElectromagneticSolverAlgo electromagnetic_solver_id, const bool fields_to_plot[], const int depos_order, const bool galerkin_interpolation, bool is_full_diagnostic );
```

Comment:
```suggestion
    ElectromagneticSolverAlgo electromagnetic_solver_id,  bool fields_to_plot[],  int depos_order,  bool galerkin_interpolation, bool is_full_diagnostic );
```

There is no need for `const` here, since these parameters are passed by value ( `const` is only meaningful in the corresponding `.cpp` file). I would suggest removing it.

Reply:
changing this raises an "invalid `const bool*` to `bool*` conversion"